src/content/blam/tags/bitmap/readme.md

```diff
@@ -125,35 +113,40 @@ Used mostly for special effects this setting stores the rgb channels as directio
 
 # 3D Textures
 3D textures are sets of textures (slices) that make up a solid 3D cube. They are only used in shader_transparent_plasma tags, which are used for the energy shield effect for instance.
 
 ### Importing
-3D textures need to be imported the same way as you import a [[Bitmap#Sprites|sprite]] sheet, all slices should be put next to each other horizontally. Preferably you want to have the same amount of slices as the height and width of each slice. So if each slice is 64x64, you want 64 slices to make it into a 64x64x64 cube.
+3D textures need to be imported the same way as you import a [sprite](#sprites) sheet, all slices should be put next to each other horizontally. Preferably you want to have the same amount of slices as the height and width of each slice. So if each slice is 64x64, you want 64 slices to make it into a 64x64x64 cube.
 
 # Cube maps
 Cube maps are a combination of 6 textures that form one cube. It is used for reflections in shaders.
 
 ### Importing
-Cube maps are imported either as folded out cubes or [[Bitmap#Sprites|sprite]] sheets. All faces should have the same resolution, power-of-two and square. Just like [[Bitmap#Sprites|sprites]], cube map bitmaps can contain multiple sequences/cube maps for use with forced permutations.
+Cube maps are imported either as folded out cubes or [sprite](#sprites) sheets. All faces should have the same resolution, power-of-two and square. Just like [sprites](#sprites), cube map bitmaps can contain multiple sequences/cube maps for use with forced permutations.
 
 _NOTE: Having multiple cube maps in one bitmap does not randomize them._
 
-#### Examples of acceptable source images for cube map import:
+### Examples of acceptable source images for cube map import:
 
-[Images still need to be uploaded]
+|Unfolded|Sprite Sequence|Multiple in one tag|
+|--------|------------|-------------------|
+<img src="media/bitmap_cubemap_swamp.png" width="200">|<img src="media/bitmap_cubemap_beach.png" width="200"><br><br><img src="media/bitmap_cubemap_grunt.png" width="200">|<img src="media/bitmap_cubemap_elite_multiple.png" width="200">|
 
 # Sprites
 
-[Images still need to be uploaded]
+<img src="media/bitmap_sprites_source_vs_tag.png" width="600">
 
-The sprite type allows a bitmap to contain a non-power-of-two texture, with support for animations with multiple permutations (sequences). Sprites are typically used for particles.
+The sprite type allows a bitmap to contain a non-power-of-two texture, with support for animations with multiple permutations (sequences). Sprites are typically used for particles.  
 
 ### Color Plate
 
-[Images still need to be uploaded]
+|Scaled 10x | Original Size |
+|-----------|---------------|
+![](media/bitmap_color_plate.png)|![](media/bitmap_color_plate_orig_size.png)|
```

Comment:
Images by default are styled as blocks with 100% width. That means this orig_size one is being shown scaled up and blurry. We need a solution here for image sizing and alignment within the content. E.g. should all images take up 100% width of their containers? Only when used in figures and metaboxes? What kind of standard image sizes do we want to support?

Doesn't have to be solved in this PR, but thought I'd point out that this image could use a style override in HTML as a workaround for now.

Reply:
Yeah, this doesn't happen in github markdown. Which is what this version of the document was made for

Reply:
A style override would be a good thing here.